lib/etl/load/create_new_group_records.rb

```diff
@@ -0,0 +1,32 @@
+class CreateNewGroupRecords
+  def write(attributes_list)
+    @attributes_list = attributes_list
+
+    attributes_list.each do |attributes|
+      puts "#{attributes["name"]}"
+      
+      if existing_record attributes
+        ap "Name already in database: #{name attributes}" # rubocop:disable Rails/Output
+      else
+        record = Group.create!(attributes)
+
+        ap "Name: #{record.name}" # rubocop:disable Rails/Output
```

Comment:
I'm wondering, do we really need this? I removed them from the other load class because Rubocop wasn't raising any offense anyway. Did you get the offenses without the comments?